RallyHereIntegration/Config/BaseRallyHereIntegration.ini

```diff
@@ -109,8 +109,8 @@ DefaultPollingInterval = 60.0
 +PollingIntervals=(TimerName="PlayerPresence",Interval=60.0)
 +PollingIntervals=(TimerName="PlayerSessions",Interval=60.0)
 +PollingIntervals=(TimerName="PlayerMatches",Interval=600.0)
-+PollingIntervals=(TimerName="FriendsList",Interval=30.0)
-+PollingIntervals=(TimerName="BlockedPlayersList",Interval=30.0)
++PollingIntervals=(TimerName="FriendsList",Interval=120.0)
```

Comment:
Lengthen intervals for rarely changed data, and rely on notifications instead for fast response.